ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Run.hs

```diff
@@ -56,16 +56,17 @@ analyse DBAnalyserConfig{analysis, confLimit, dbDir, selectDB, validation, verbo
         mkProtocolInfo args
       let chunkInfo  = Node.nodeImmutableDbChunkInfo (configStorage cfg)
           args' =
-            Node.mkChainDbArgs
-              registry InFuture.dontCheck cfg genesisLedger chunkInfo $
-            ChainDB.defaultArgs (Node.stdMkChainDbHasFS dbDir)
-          chainDbArgs = args' {
-              ChainDB.cdbImmutableDbValidation = immValidationPolicy
-            , ChainDB.cdbVolatileDbValidation  = volValidationPolicy
-            , ChainDB.cdbTracer                = chainDBTracer
-            }
-          (immutableDbArgs, _, _, _) = fromChainDbArgs chainDbArgs
-          ledgerDbFS = ChainDB.cdbHasFSLgrDB chainDbArgs
+            completeChainDbArgs
+              registry
+              InFuture.dontCheck
+              cfg
+              genesisLedger
+              chunkInfo
+              (const True)
```

Comment:
Seems slightly unergonomic to first set this to a placeholder value and then changing it using `maybeValidateAll`. Could we work around that?

Reply:
These are two different things. This argument is `(imm|vol)checkIntegrity` and the one you seem to be thinking of is `(imm|vol)ValidationPolicy`

Reply:
Oops, thanks for pointing that out!